src/sql/magic_plot.py

```diff
@@ -72,6 +79,15 @@ def execute(self, line="", cell="", local_ns=None):
         else:
             column = cmd.args.column
 
+        bin_specified = False
+        pattern = r"histogram.*( --bins | -b )"
+        if re.search(pattern, line):
+            bin_specified = True
+            if re.search(r"( --breaks | -B )", line):
+                raise exceptions.UsageError(
+                    "Both bins and breaks are specified. Must specify only one of them."
+                )
+
```

Comment:
I think you can use `cmd` to check if both arguments exist.

e.g.,

```python
if cmd.args.bins and cmd.args.breaks:
    raise UsageError(...)
```

let me know if that works, otherwise we should fix it since validating arguments with regular expressions is going to be a pain if we have more complex combinations

Reply:
Because of the default value of bins (which is 50 for the magic command), `cmd.args.bins` always return an integer value. Thus, I used the following code instead.

```
bin_specified = " --bins " in line or " -b " in line
breaks_specified = " --breaks " in line or " -B " in line
if bin_specified and breaks_specified:
     raise exceptions.UsageError(...)
```